spec/models/searchgov_domain_spec.rb

```diff
@@ -128,7 +128,7 @@
 
       describe '.not_ok' do
         it 'includes inaccessible domains' do
-          expect(described_class.not_ok).to match_array [not_ok_domain]
+          expect(described_class.not_ok).to include not_ok_domain
```

Comment:
This spec is no longer valid, as it would pass if the scope included every domain. Be very careful not to reduce the functionality of existing specs when making changes. We need one or more specs that will verify that:
- a domain with a "200 OK" status is not included in the scope (which this spec used to do, but it no longer does with your most recent change)
- a domain with a status other than `200 OK` is included in this scope
- a domain with a nil status is included in the scope